components/table/index.css

```diff
@@ -608,48 +482,55 @@
 		border-end-end-radius: var(--mod-table-border-radius, var(--spectrum-table-border-radius));
 	}
 
-	&:focus {
-		outline: 0;
-	}
-
-	&:hover,
+	/* Row focus indicator */
+	&.is-focused,
 	&:focus-visible,
-	&.is-focused {
-		--highcontrast-table-row-text-color: var(--highcontrast-table-row-text-color-hover);
-		--highcontrast-table-icon-color: var(--highcontrast-table-row-text-color-hover);
-		--spectrum-table-cell-background-color: var(--highcontrast-table-row-background-color-hover, var(--mod-table-row-background-color-hover, var(--spectrum-table-row-background-color-hover)));
-	}
+	&:focus {
+		outline: none;
 
-	&:active {
-		--highcontrast-table-row-text-color: var(--highcontrast-table-row-text-color-hover);
-		--highcontrast-table-icon-color: var(--highcontrast-table-row-text-color-hover);
-		--spectrum-table-cell-background-color: var(--highcontrast-table-row-background-color-hover, var(--mod-table-row-active-color, var(--spectrum-table-row-active-color)));
-	}
+		/* First cell of focused rows need fancy new focus indicator borders & no border-radius, and the thicker row focus indicator */
```

Comment:
This was pretty complex, so again- feedback is welcome! 

The table row itself can be focused.
It's got a 4px blue side focus indicator, as well as (basically) a border that goes around the entire row. 
Some rows don't have border radii, and some do. But- those border radii are created with the individual table cells in that row.

🥵🥵🥵

Reply:
I sent a message to Stefan to clarify the "borders around a focused row." That _might_ just be for an emphasized table? I need some clarity on that.

Reply:
Looks like you added a different TODO comment re: the shift?